TestGMEOffers.py

```diff
@@ -0,0 +1,25 @@
+from Artesian import GMEPublicOfferService
+from Artesian import ArtesianConfig
+from Artesian import Market
```

Comment:
I think these enums should be somehow scoped under GMEPublicOffer

I'd expect to write `from Artesian.GMEPublicOffer import stuff`, top level Status or Scope doesn't look right in a Python library.